source/Octopus.Client/Model/LicenseStatusResource.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Octopus.Client.Serialization;
+
+namespace Octopus.Client.Model
+{
+    public static class LicenseKinds
+    {
+        public static readonly string Unlicensed = "Unlicensed";
+        public static readonly string Trial = "Trial";
+        public static readonly string CommunityEdition = "CommunityEdition";
+        public static readonly string CommercialLicense = "CommercialLicense";
+    }
+    
+    public static class LicenseMessageDispositions
```

Comment:
This is an enum on the server side. Are you using `String`s here instead of the enum to avoid problems in the future when the enum gets extended ?

Reply:
That is right